pkg/3scale/amp/component/system.go

```diff
@@ -92,11 +93,24 @@ const (
 )
 
 const (
-	SystemSidekiqName = "system-sidekiq"
+	SystemSidekiqName       = "system-sidekiq"
+	SystemAppDeploymentName = "system-app"
+
+	SystemAppMasterContainerName    = "system-master"
+	SystemAppProviderContainerName  = "system-provider"
+	SystemAppDeveloperContainerName = "system-developer"
 )
 
 const (
-	SystemSidekiqMetricsPort = 9394
+	SystemAppPrometheusExporterPortEnvVarName     = "PROMETHEUS_EXPORTER_PORT"
+	SystemSidekiqPrometheusExporterPortEnvVarName = "PROMETHEUS_EXPORTER_PORT"
+	SystemSidekiqMetricsPort                      = 9394
+	SystemAppMasterContainerPrometheusPort        = 9394
+	SystemAppProviderContainerPrometheusPort      = 9395
+	SystemAppDeveloperContainerPrometheusPort     = 9396
```

Comment:
AFAIK these ports don't match what we have in the System code 😕 
https://github.com/3scale/porta/blob/9c175735a7ada82e33fc90cab4a8b762c8a7ad9d/lib/prometheus_exporter_port.rb#L5-L10

Reply:
Do not have to match. System code defines default ports if not set. We have decided to set them in order to know exactly port numbers. As we do for other components as well.

Is it an issue? Do those port numbers get in conflict with other services in the same pod?

Reply:
@hallelujah should say. I do not know about this, but I only remember that when we reviewed his PR in Porta/System, he told us that those ports numbers came from an agreement made with you guys. And later those changed for an issue. So... if they changed in System, why not here? if they do not need to match, why did we have to agree in the port numbers in the first place?

Reply:
Because we need those default ports to be different for templates. In templates ports are not set, monitoring only enabled in operator based deployments.

Reply:
I think it's better to have them match though, so between templates and operator the used ports are effectively the same.

I'll change it to match the new values if there's no objection.

Reply:
Updated to the new values